### PR TITLE
textures: test pvr & dds

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: modules/textures/test/lib/parsers/parse-dds.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    format: GL.COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL
+  }
+];
+
+test('CompressedTextureLoader#dds', async t => {
+  for (const testCase of TEST_CASES) {
+    const response = await fetchFile(testCase.url);
+    const data = await response.arrayBuffer();
+    const result = parseDDS(data);
+    t.ok(result);
```
Comment (ibgreen) [inline modules/textures/test/lib/parsers/parse-dds.spec.js:39]: I think can drop the `t.ok`

File: modules/textures/test/lib/parsers/parse-pvr.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    format: GL.COMPRESSED_SRGB8_ALPHA8_ASTC_10X10_KHR
+  }
+];
+
+test('CompressedTextureLoader#pvr', async t => {
+  for (const testCase of TEST_CASES) {
+    const response = await fetchFile(testCase.url);
+    const data = await response.arrayBuffer();
+    const result = parsePVR(data);
+    t.ok(result);
```
Comment (ibgreen) [inline modules/textures/test/lib/parsers/parse-pvr.spec.js:39]: ditto

File: modules/textures/test/test-utils/check-compressed-texture.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export function checkCompressedTexture(imageData, testCase, t) {
```
Comment (ibgreen) [inline modules/textures/test/test-utils/check-compressed-texture.js:1]: I usually put `t` parameter first in these functions.

**Review by ibgreen — APPROVED** [review]

File: modules/textures/test/lib/parsers/parse-dds.spec.js (hunk 1 above)
Comment (belom88) [reply modules/textures/test/lib/parsers/parse-dds.spec.js]: Done

File: modules/textures/test/lib/parsers/parse-pvr.spec.js (hunk 2 above)
Comment (belom88) [reply modules/textures/test/lib/parsers/parse-pvr.spec.js]: Done

File: modules/textures/test/test-utils/check-compressed-texture.js (hunk 3 above)
Comment (belom88) [reply modules/textures/test/test-utils/check-compressed-texture.js]: Done

**Pull request merged**